gnupg/_parsers.py

```diff
@@ -74,6 +80,67 @@ def _check_keyserver(location):
                 return keyserver
             return None
 
+
+def _check_keyserver_option(ks_option):
+    """Check that the provided keyserver option is valid and safe.
+
+    :param str ks_option: A valid argument to --keyserver-option.
+    :rtype: :obj:`str` or :obj:`None`
+    :returns: A string of the keyserver option or None.
+    """
+    def _is_valid_file(option_value):
```

Comment:
You probably want to use `gnupg._util._is_file()` here.

Reply:
Fixing.